ocrd_monitor/ocrdbrowser/_docker.py

```diff
@@ -23,6 +22,7 @@ def __init__(self, host: str, port: Port, owner: str, workspace: str) -> None:
         self._port = port
         self._owner = owner
         self._workspace = path.abspath(workspace)
+        self.id: str | None = None
```

Comment:
Is there are reason why variables names starting with underscore and some not?

Reply:
Attributes starting with an underscore are considered private by convention